src/mempools/dynamic_mempool.hpp

```diff
@@ -118,7 +118,7 @@ namespace argo {
 				 * @param a a pointer to the allocator to use for growing this mempool
 				 * @todo passing the allocator by pointer may be unnecessary
 				 */
-				dynamic_memory_pool(allocator_t* a) : allocator(a), memory(nullptr), max_size(0), offset{0} {}
+				explicit dynamic_memory_pool(allocator_t* a) : allocator(a), memory(nullptr), max_size(0), offset{0} {}
```

Comment:
Should this possibly be part of #95 instead?
Feel free to dismiss, not critical in any way.